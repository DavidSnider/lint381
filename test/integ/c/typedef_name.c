typedef struct Foo * Foo_ptr;

typedef struct Foo * Foo_ptr_t;

typedef struct {
    // Should not throw an error.
    struct Foo_t bar;
};
