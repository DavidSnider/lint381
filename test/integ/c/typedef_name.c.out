typedef_name.c:1:22: error: Typedef 'Foo_ptr' should end with '_t'
typedef struct Foo * Foo_ptr;
                     ^^^^^^^
